frontend/src/app/components/time/time.component.ts

```diff
@@ -3,6 +3,28 @@ import { StateService } from '../../services/state.service';
 import { dates } from '../../shared/i18n/dates';
 import { DatePipe } from '@angular/common';
 
+const datePipe = new DatePipe(navigator.language || 'en-US');
```

Comment:
Not sure if it's this row causing it .. But my browser just errors out with:

```
core.mjs:6531 ERROR RuntimeError: NG02100: InvalidPipeArgument: 'NG0701: Missing locale data for the locale "sv-SE".' for pipe 'DatePipe'
    at invalidPipeArgumentError (common.mjs:4053:12)
    at DatePipe.transform (common.mjs:4527:19)
    at TimeComponent.calculate (time.component.ts:126:28)
    at TimeComponent.calculateTime (time.component.ts:82:45)
    at TimeComponent.ngOnChanges (time.component.ts:73:10)
```

Reply:
this should be fixed now with a new approach.